src/Frontend/Themes/Bootstrap4/Modules/Profiles/Layout/Templates/ChangePassword.html.twig

```diff
@@ -0,0 +1,37 @@
+{% import 'Core/Layout/Templates/Alerts.html.twig' as alerts %}
+{% if updatePasswordSuccess %}
```

Comment:
die moeten er idd in, want als er form errors zijn word er gescrolled naar id op de section en dan vallen de alerst uit beeld